storage/innobase/xtrabackup/src/redo_log.cc

```diff
@@ -84,28 +86,28 @@ bool Redo_Log_Reader::find_start_checkpoint_lsn() {
 
     log_files_header_read(*log_sys, max_cp_field);
 
-    if (checkpoint_no_start !=
-        mach_read_from_8(log_sys->checkpoint_buf + LOG_CHECKPOINT_NO)) {
+    if (checkpoint_lsn_start !=
+        mach_read_from_8(log_sys->checkpoint_buf + LOG_CHECKPOINT_LSN)) {
       checkpoint_lsn_start =
           mach_read_from_8(log_sys->checkpoint_buf + LOG_CHECKPOINT_LSN);
-      checkpoint_no_start =
-          mach_read_from_8(log_sys->checkpoint_buf + LOG_CHECKPOINT_NO);
```

Comment:
I wonder if we use this checkpoint_no stuff for pre-8.0.30. @altmannmarcelo can you please confirm if you are using this or not?

Reply:
it didn't make any sense in 8.0.29 too. Better to remove the definition in  log_sys struct @altmannmarcelo